infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LRRollingUpgradeHandler.java

```diff
@@ -56,28 +58,35 @@ public LRRollingUpgradeHandler(ILogReplicationVersionAdapter versionAdapter) {
     }
 
     public boolean isLRUpgradeInProgress(TxnContext txnContext) {
-        if (isClusterAllAtV2) {
+        try {
+            txnContext.getTable(LOG_REPLICATION_PLUGIN_VERSION_TABLE);
+
```

Comment:
can you add a comment here stating that this table was removed in the current version.  So if it is found, this node has undergone an upgrade from a previous version.